app/components/job/details/error-display/job-error-display.html

```diff
@@ -0,0 +1,16 @@
+<bex-banner *ngIf="jobFailed" fixMessage="View failed tasks" [fix]="gotoFailedTask" type="warning">
+    <div message>Job was terminated because one task failed.</div>
```

Comment:
should we say that "one task failed", or just that "a task failed". The first option might be confusing to users if in fact more than one task failed.